app/bundles/PluginBundle/Integration/AbstractIntegration.php

```diff
@@ -2752,6 +2757,7 @@ public function getLeadDoNotContact($leadId, $channel = 'email')
     public function getCompoundMauticFields($lead)
     {
         if ($lead['internal_entity_id']) {
+            $lead['mauticContactId']                   = $lead['id'];
```

Comment:
Should be $lead['internal_entity_id'] to remove question of ambiguity (just like the lines below). Getting the beta release out so moving this to 2.14.1